### ISSUE TITLE
Some questions about Focal Convolution Layers (FConv)

### ISSUE THREAD

**Issue by ShinanZou** [ChaoFan96/GaitPart#5]
May I ask whether activation and pooling are performed on each of the segmented feature maps respectively, or after the segmented feature maps are merged?I think there is no difference here.What do you think?

**Comment by ChaoFan96** [comment]
Thanks for your attention!
In this work, the operations of pooling as well as activation are both performed on the output feature map of the FConv, and in your word, they are performed after those split feature maps being concatenated.
Hope this repsond could help you~

**Comment by ShinanZou** [comment]
Hi, I want to know if FConv is applying a padding to the entire feature or applying a padding to each piece that is cut horizontally

**Comment by ChaoFan96** [comment]
In our implementations, the FConv works in the latter way.

**Comment by ShinanZou** [comment]
Hi, can you tell me how you initialize network parameters in your implementation?And what frameworks are used?Pytorch?

**Comment by ChaoFan96** [comment]
Yes, it's totally a PyTorch implementation and all the parameters (including Conv's &FC's) are initialized by 'torch.nn.init.xavier_uniform_'.

**Comment by ShinanZou** [comment]
May I ask: in Fconv, "sharing the same kernel" refers to the size of the core or the same weight?Is there any bias in convolution？

**Comment by ChaoFan96** [comment]
Hello, there is only one convolution kernel skip among the split feature map.
And all the layers, including convolution and fully-connected layers, are bias-free.

**Comment by zsn-life** [comment]
So what's the pytorch version?

**Comment by ShinanZou** [comment]
So what is the Pytorch version in your implementation?

**Comment by ChaoFan96** [comment]
Hello, I think it may be 1.0.0

**Comment by ChaoFan96** [comment]
Hi, the OpenGait is released now! ( https://github.com/ShiqiYu/OpenGait )
This project not only contains the full code of gaitpart but also reproduces several SOTA models of gait recognition.
Enjoy it and any questions or suggestions are welcome!
